lg_media/scripts/image_viewer.py

```diff
@@ -155,13 +177,18 @@ def _create_pqiv(self, image):
         r = requests.get(image.url)
         with open(image_path, 'wb') as f:
             f.write(r.content)
-        command = '/usr/bin/pqiv -c -i --scale-mode-screen-fraction=1.0 -T {} -P {},{} {}'.format(
+        opts = '-t'
+        if graphic_opts[(image.url, image.geometry.x, image.geometry.y)]['no_upscale']:
```

Comment:
does using a `.get('no_upscale', False)` make sense here to avoid accessing a dictionary

Reply:
Yes, I will change it, that will be safer, they are both declared when reading the message but in a different function, who knows what could go wrong